lib/config.ts

```diff
@@ -154,7 +154,6 @@ export type Config = {
         username?: string;
         password?: string;
         bearertoken?: string;
-        iap_receipt?: string;
```

Comment:
https://github.com/DIYgod/RSSHub/blob/2a2a3066bfbc9d59e0632bff5575d960bd13a7b9/lib/config.ts#L508-L513
is still not being removed

Reply:
Nice catch! My bad